### PR TITLE
[Events] Use old layout for non-live events and Add chat watcher count

### PR DESCRIPTION
**Pull request by rkdavidson**
# Related Issue
- [**🏕 [Events] Remove schedule from Live event layout**](https://3.basecamp.com/3926363/buckets/17566548/todos/3094801989)
- [**🏕 [Events] Use existing layout for non-live events**](https://3.basecamp.com/3926363/buckets/17566548/todos/3076462962)
- [**🏕 [Chat] Show watcher count**](https://3.basecamp.com/3926363/buckets/17566548/todos/3100735047)

# Changes or Updates
- Cleans up component structure so the old Events layout is rendered with the new components for non-live events
- Pulls **Schedule** info out of side-panel, and puts it down by the description (for live events)
- Adds number of users currently watching

# Test Instructions
- Open an event that is not live locally _and_ on production site
  - I.e. http://local.christfellowship.church:3000/events/tune-in-online and https://christfellowship.church/events/tune-in-online
- 🔸 Verify the layouts closely match, across screen sizes (note: some typography tweaks were made within Schedule display)
- Open an event that is live
- 🔸 Verify:
  - The **Schedule** info is no longer in the side panel, but down by the description area
  - The number of users currently watching is displayed in the top right... log in as a different user in an Incognito window and refresh them to validate decrement/increment as connections change
  - Gut-check responsive layout sizes

# Screenshots

|Non-Live Event|Live Event|
|---|---|
|||
|CleanShot 2020-10-09 at 10 54 20@2x|CleanShot 2020-10-09 at 11 39 31@2x|

|Watcher Count (recorded before design polish)|
|---|
||

# Tests
- [ ] needed snapshots updated
- [ ] mobile tested
- [ ] latest update from master

### CODE REVIEW

**Review by jordanwade — CHANGES_REQUESTED** [review]

**Comment by jordanwade** [comment]
@rkdavidson 
Looking good, just saw a handful of console logs and one issue with the the watch count.

**Comment by jordanwade** [comment]
Also, just saw a small issue with an empty card on the `wake` event. Ignore if it isn't related to your changes.

image

**Comment by rkdavidson** [comment]
Good catch on that Wake event, @jordanwade — not sure how to gracefully handle this, other than returning null so the whole description slots over to the left in its place.

This is some of the stuff we'll circle back with and fix scheduling overall for, so hopefully a temporary problem.

File: src/content-single/EventContentItem/components/EventSchedule.js [hunk 1]
```diff
@@ -4,22 +4,24 @@ import classnames from 'classnames';
...
 
-import EventScheduleTimes from './EventScheduleTimes';
+import CallsToAction from './CallsToAction';
 import CampusSelector from './CampusSelector';
+import EventScheduleTimes from './EventScheduleTimes';
 
 function getScheduleByLocation(events) {
-  // console.groupCollapsed('[schedule] 📍 getScheduleByLocation()');
-  // console.log('[schedule] events:', events);
+  console.groupCollapsed('[schedule] 📍 getScheduleByLocation()');
```
[the remark this answers is not in the document]
Comment (rkdavidson) [reply src/content-single/EventContentItem/components/EventSchedule.js:14]: No, probably not anymore — meant to remove them but added back to triage issue earlier today. Will remove.

**Review by jordanwade — APPROVED** [review]

File: src/content-single/EventContentItem/components/CallsToAction.js [hunk 2]
```diff
@@ -4,46 +4,37 @@ import { isEmpty } from 'lodash';
 
 import { GoogleAnalytics } from 'analytics';
 
-import { Row, Col } from 'ui';
-
-const CallsToAction = ({ eventTitle, items }) => {
+const CallsToAction = ({ eventTitle, items, hasEvents }) => {
```
[the remark this answers is not in the document]
Comment (rkdavidson) [reply src/content-single/EventContentItem/components/CallsToAction.js:7]: Great catch, @jordanwade — the product requirements here are in flux/fuzzy, but to keep existing behavior, I updated the new `EventSchedule` to properly pass `hasEvents` down to conditionally show the **Get Started** subhead.

Insert spiel about how we'll imminently revisit this **Schedule** and **Calls to Action** portion, regardless 🙃.

Commit: 4445e5a

File: src/content-single/EventContentItem/components/EventChat.js [hunk 3]
```diff
@@ -129,24 +130,36 @@ const EventChat = ({ event, channelId }) => {
...
-          console.log('[dm] --> event: ', event);
+          console.log('[dm] --> clientEvent: ', clientEvent);
           getDmChannels();
         }
 
         // To prevent stale state data, and avoid having to useRef() etc, we'll start
         // doing a state update on activeDmChannel but return the current value.
         return currentActiveDmChannel;
       });
+    } else if (
```
[the remark this answers is not in the document]
Comment (rkdavidson) [reply src/content-single/EventContentItem/components/EventChat.js]: Great catch — updated in 11a5dd7 to count events related to _guests_ OR users other than me.

**Comment by rkdavidson** [comment]
@jordanwade Responded to your feedback in a series of small commits, and addressed the empty schedule area on that Wake event by returning `null` if there are no events _and_ no calls to action — i.e. nothing to render.

**Comment by rkdavidson** [comment]
@jordanwade I went ahead and merged this to minimize churn/conflicts with the **Stream XP** work.
I'm confident in the changes/fixes for the items you called out 🤞.

**Pull request merged**